app/src/main/java/de/westnordost/streetcomplete/quests/bus_stop_type/AddBusStopType.java

```diff
@@ -0,0 +1,52 @@
+package de.westnordost.streetcomplete.quests.bus_stop_type;
+
+import android.os.Bundle;
+
+import de.westnordost.streetcomplete.data.QuestImportance;
+import de.westnordost.streetcomplete.data.osm.OverpassQuestType;
+import de.westnordost.streetcomplete.data.osm.changes.StringMapChangesBuilder;
+import de.westnordost.streetcomplete.quests.AbstractQuestAnswerFragment;
+
+public class AddBusStopType extends OverpassQuestType
+{
+	@Override
+	protected String getTagFilters()
+	{
+		return "nodes with public_transport=platform and !shelter and !informal";
```

Comment:
Makes sense to support `highway=bus_stop` as well.
`informal=yes` is documented on the wiki mainly to use with footways to distinguish "desire paths" (paths created by cutting on the grass), but the page says it can be extended for other uses. Informal bus stops are working by local knowledge, for example school bus is known by local children to stop here. However as you said it not can always be seen on the ground.

Maybe, to make the quest more interesting, we can ask for a combination of shelter + bench.  We can show 4 pictures and ask what the stop looks like:
- No shelter and no bench
- No shelter but a bench
- Shelter without bench
- Shelter and bench

What do you think ?